packages/apps/staking/src/theme.ts

```diff
@@ -1,359 +1,366 @@
-    },
-    info: {
-      main: colorPalette.info.main,
-      light: colorPalette.info.light,
-      dark: colorPalette.info.dark,
-    },
-    secondary: {
-      main: colorPalette.secondary.main,
-      light: colorPalette.secondary.light,
-    },
-    text: {
-      primary: colorPalette.primary.main,
-      secondary: colorPalette.fog.main,
-    },
-    sky: {
-      main: colorPalette.sky.main,
-      light: colorPalette.sky.light,
-      dark: colorPalette.sky.dark,
-      contrastText: colorPalette.sky.contrastText,
-    },
-    white: {
-      main: '#fff',
-      light: '#fff',
-      dark: '#fff',
-      contrastText: '#fff',
-    },
-    textSecondary: colorPalette.textSecondary,
-  },
-  typography: {
-    fontFamily: 'Inter, Arial, sans-serif',
-    h1: {
-      fontSize: 32,
-      fontWeight: 600,
-    },
-    h2: {
-      fontSize: 34,
-      fontWeight: 600,
-    },
-    h3: {
-      fontSize: 24,
-      fontWeight: 500,
-    },
-    h4: {
-      fontSize: 20,
-      fontWeight: 500,
-    },
-    h5: {
-      fontSize: 18,
-      fontWeight: 600,
-    },
-    h6: {
-      fontSize: 20,
-      fontWeight: 500,
-    },
-    'H6-Mobile': {
-      fontSize: '20px',
-      fontWeight: 500,
-      lineHeight: '32px',
-      letterSpacing: '0.15px',
-      textAlign: 'left',
-    },
-    body1: {
-      fontSize: 16,
-      fontWeight: 400,
-    },
-    body2: {
-      fontSize: 14,
-      fontWeight: 500,
-    },
-    body3: {
-      fontSize: '12px',
-      fontWeight: 400,
-      lineHeight: '19.92px',
-      letterSpacing: '0.4px',
-      textAlign: 'left',
-    },
-    'Components/Button Small': {
-      fontSize: '13px',
-      fontWeight: 600,
-      lineHeight: '22px',
-      letterSpacing: '0.1px',
-      textAlign: 'left',
-    },
-    'Components/Button Large': {
-      fontSize: '15px',
-      fontWeight: 600,
-      lineHeight: '26px',
-      letterSpacing: '0.1px',
-      textAlign: 'left',
-    },
-    'Components/Chip': {
-      fontSize: '13px',
-      fontWeight: 400,
-      lineHeight: '18px',
-      letterSpacing: '0.16px',
-      textAlign: 'left',
-    },
-    'Components/Table Header': {
-      fontFamily: 'Roboto',
-      fontSize: '14px',
-      fontWeight: 500,
-      lineHeight: '24px',
-      letterSpacing: '0.17px',
-      textAlign: 'left',
-    },
-    subtitle1: {
-      fontSize: 12,
-    },
-    subtitle2: {
-      fontSize: 14,
-      fontWeight: 600,
-      lineHeight: '21.9px',
-    },
-    caption: {
-      fontSize: 10,
-    },
-  },
-  components: {
-    MuiButton: {
-      styleOverrides: {
-        root: {
-          cursor: 'pointer',
-          fontWeight: 600,
-          textTransform: 'none',
-
-          '&:disabled': {
-            backgroundColor: '#FBFBFE',
-            color: 'rgba(203, 207, 232, 0.86)',
-            border: 'none',
-          },
-        },
+export const createAppTheme = (mode: 'light' | 'dark') => {
+  return createTheme({
+    palette: {
+      mode,
```

Comment:
Do we need to have `mode` here? Maybe move it one level up if possible?
```
mode,
paletter: {
....
```
Also you can declare `isLight = mode === 'light'` and reuse it below, so it is easier to search for all dependent places in the theme if needed